mbed_devices/_internal/htm_file.py

```diff
@@ -0,0 +1,63 @@
+"""Facilitates parsing MBED.HTM files found on the Mbed devices.
+
+For legacy reasons, we support many flavours of MBED.HTM files.
```

Comment:
I'd like to add some examples of actual file contents here, thoughts?

Reply:
Would be useful to have some examples of the various "flavours"

Reply:
Right, I'll need to get my hands on some devices in the office to get a decent set of examples here. 👍